model/scratch/dffml-model-anomalydetection/dffml_model_anomalydetection/anomalydetection.py

```diff
@@ -0,0 +1,247 @@
+import math
+import pathlib
+import statistics
+from typing import AsyncIterator, Type
+
+import numpy as np
+from dffml import (
+    Accuracy,
+    Feature,
+    Features,
+    ModelNotTrained,
+    Record,
+    SimpleModel,
+    SourcesContext,
+    config,
+    entrypoint,
+    field,
+)
+
```

Comment:
Let's make sure we find out if there is any source and it's licensing we might want to attribute some of this code / ideas being from